src/components/Table/Table.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react';
+import { Person } from '../../types';
+import { Row } from '../Row/Row';
+
+type Props = {
+  persons: Person[]
+};
+
+export const Table: React.FC<Props> = ({
+  persons,
+}) => {
+  const [selectedPersonSlug, setSelectedPersonSlug] = useState<string>();
+
+  const personNamesFromSlug = persons.map(currPerson => currPerson.slug
```

Comment:
not fixed 
![image](https://github.com/mate-academy/react_people-table-basics/assets/70009205/61314c2a-9423-4128-8d7c-4789a8f716ef)

Reply:
interesting I fixed all bugs. I try push one more